ui/desktop/app/components/reveal/index.hbs

```diff
@@ -0,0 +1,14 @@
+<details ...attributes class='reveal' {{on-click-outside this.close}}>
+  <summary class='reveal-summary'>
+    <span class='reveal-summary-text-close'>
+      {{@textWhenClose}}
+    </span>
+    <span class='reveal-summary-text-open'>
+      {{@textWhenOpen}}
+    </span>
+
+  </summary>
+  <content class='reveal-content' {{on-click-inside this.close}}>
```

Comment:
This is one thing that differs from dropdown.  The reveal component should probably remain open until the user explicitly closes it.  There could be important information revealed they want to copy/paste or even interact with.  So we don't need either `on-click-outside` or `-inside` modifiers.

Reply:
Awesome! yeah Iw as wondering about this behaviour will change it.